Structure_oM/MaterialFragments/LaminatedVeneerLumberCrossbands.cs

```diff
@@ -0,0 +1,213 @@
+                             "X - Shear Modulus in the local xy-plane (Gxy). For most cases this will Parallel shear modulus (G_0) - For flatwise use this will be G_0_Edge, for Edgewise use this will be G_0_Flat.\n" +
+                             "Y - Shear Modulus in the local yz-plane (Gyz). For most cases this will be the Perpendicular shear modulus - For both flatwise and edgewise G_90_flat.\n" +
+                             "Z - Shear Modulus in the local zx-plane (Gzx). For most cases this will Parallel shear modulus (G_0) - For flatwise use this will be G_0_Flat, for Edgewise use this will be G_0_Edge.")]
+        public virtual Vector ShearModulus { get; set; }
+
+        [Ratio]
+        [Description("Poisson's Ratio. Ratio between axial and transverse strain. Typically taken as 0.4 for X and Y component (νxy and νyz) and as 0.4*E_90/E_0 for the Z component, though value varies depending on timber species.\n" +
+                     "Vector components made up of:\n" +
+                     "X - Poisson's ratio for strain in the local y direction generated by unit strain in x direction (νxy). Generally strain in perpendicular direction caused by strain in longitudinal direction.\n" +
+                     "Y - Poisson's ratio for strain in the local z direction generated by unit strain in y direction (νyz). Generally strain in perpendicular direction caused by strain in other perpendicular direction.\n" +
+                     "Z - Poisson's ratio for strain in the local x direction generated by unit strain in z direction (νzx). Generally strain in longitudinal direction caused by strain in perpendicular direction. Note that this value generally is significantly lower than values for the other two components.")]
+        public virtual Vector PoissonsRatio { get; set; }
+
+        [ThermalExpansionCoefficient]
+        [Description("Thermal Expansion Coefficeint. Strain induced in the material per unit change of temperature. Typically take as 5x10^-6 in all directions, though value varies depending on timber species.\n" +
+                     "Vector defines stiffnesses as follows:\n" +
+                     "X - Thermal expansion along the local x-axis of the element (αx).\n" +
+                     "Y - Thermal expansion along the local y-axis of the element (αy).\n" +
+                     "Z - Thermal expansion along the local z-axis of the element (αz).")]
+        public virtual Vector ThermalExpansionCoeff { get; set; }
+
+
+        /***************************************************/
+        /**** Properties - Stiffness                    ****/
+        /***************************************************/
+
+        [YoungsModulus]
+        [Description("Characteristic modulus of elasticity parallel to grain, E0,k in most manufacturer documentation. Value same for Em,0,edge,k, Em,0,flat,k, Et,0,k, and Ec,0,k.")]
+        public virtual double E_0_k { get; set; }
```

Comment:
```suggestion
        [Description("Characteristic modulus of elasticity parallel to grain, E0,k in most manufacturer documentation. Value is the same for Em_0_edge_k, Em_0_flat_k, Et_0_k, and Ec_0_k.")]
        public virtual double E_0_k { get; set; }
```

Are these supposed to be underscores - there's quite a few instances below.

Reply:
Used ',' in the descriptions where I could (matches how it is noted in documentation and Europcode), and underscored where I couldn't (proeprty names). Again, good to chat through I think